tests/test_target.py

```diff
@@ -64,7 +64,7 @@ class B(Driver):
     target.activate(a2)
     with pytest.raises(NoDriverFoundError) as excinfo:
         target[A]
-    assert "multiple active drivers" in excinfo.value.msg
+    assert "multiple drivers" in excinfo.value.msg
```

Comment:
Why is this change still needed?

Reply:
I missed the multiple drivers line, good catch.